draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1551,76 +1555,92 @@ user agent might wish to block sending cookies during "third-party" requests
+### Retrieval Algorithm {#retrieval-algorithm}
 
-        Or:
+The user agent MUST use an algorithm equivalent to the following algorithm to
+compute the cookie-string from a cookie store, a retrieval-uri, and the retrieval's
+same-site status.
+
+1. Let cookie-list be the set of cookies from the cookie store that meets all
+   of the following requirements:
+
+   * Either:
 
-        *   The cookie's host-only-flag is false and the canonicalized
-            request-host domain-matches the cookie's domain.
+     *   The cookie's host-only-flag is true and the canonicalized
+         host of the retrieval-uri is identical to the cookie's domain.
 
-    *   The request-uri's path path-matches the cookie's path.
+     Or:
 
-    *  If the cookie's secure-only-flag is true, then the request-uri's
-       scheme must denote a "secure" protocol (as defined by the user agent).
+     *   The cookie's host-only-flag is false and the canonicalized
+         host of the retrieval-uri domain-matches the cookie's domain.
 
-       NOTE: The notion of a "secure" protocol is not defined by this document.
-       Typically, user agents consider a protocol secure if the protocol makes
-       use of transport-layer security, such as SSL or TLS. For example, most
-       user agents consider "https" to be a scheme that denotes a secure
-       protocol.
+   * The retrieval-uri's path path-matches the cookie's path.
 
-    *  If the cookie's http-only-flag is true, then exclude the cookie if the
-       cookie-string is being generated for a "non-HTTP" API (as defined by
-       the user agent).
+   * If the cookie's secure-only-flag is true, then the retrieval-uri's
+     scheme must denote a "secure" protocol (as defined by the user agent).
 
-    *  If the cookie's same-site-flag is not "None", and the HTTP request is
-       cross-site (as defined in {{same-site-requests}}) then exclude the
-       cookie unless all of the following statements hold:
+     NOTE: The notion of a "secure" protocol is not defined by this document.
+     Typically, user agents consider a protocol secure if the protocol makes
+     use of transport-layer security, such as SSL or TLS. For example, most
+     user agents consider "https" to be a scheme that denotes a secure
+     protocol.
 
-        1.  The same-site-flag is "Lax" or "Default".
+   * If the cookie's http-only-flag is true, then exclude the cookie if the
+     cookie-string is being generated for a "non-HTTP" API.
 
-        2.  The HTTP request's method is "safe".
+   * If the cookie's same-site-flag is not "None" and the retrieval's same-site
+     status is "cross-site", then exclude the cookie unless all of the following
+     conditions are met:
 
-        3.  The HTTP request's target browsing context is a top-level browsing
-            context.
+     * The retrieval is for a Cookie header in the HTTP request associated with
+       the retrieval-uri The same-site-flag is "Lax" or "Default".
```

Comment:
(nit: it looks like this should be 2 lines)

Reply:
Good catch; I auto-reformatted to 80 characters in vim and that didn't respect markdown bullets. Fixed.